app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="historical_graph">Historical Graph</string>
+    <string name="widget_graph_we_are_retrieving_data">We are retrieving some data, please came back later</string>
```

Comment:
Is this something that take several minutes? Otherwise we can change the text to something that does not give the impression it will take long.

Reply:
I'll look into this usually depends on how the sensor get refreshed. For example if is electric sensor the W will change very often but it is a for example temperature of a room it might not change that often.

But I can change it for something more generic